web/.editorconfig

```diff
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.ts]
+quote_type = single
+
+[*.md]
+max_line_length = off
+trim_trailing_whitespace = false
```

Comment:
Shall we also add `insert_final_newline = false` for .md files?

Reply:
Why do we want `insert_final_newline` set to `false`?

Reply:
Ah, I think I miss read it.